hrui/CRM/HRUI/Helper.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Collection of methods to be used in this extension
+ */
+class CRM_HRUI_Helper {
+
+  /**
+   * @param int $contactID contact ID
+   * @return string|null return comma separated string containing list of line managers
+   * for specific contact
+   */
+  public static function getLineManagersList($contactID)  {
+    try {
+      $result = civicrm_api3('RelationshipType', 'getsingle', array(
+        'sequential' => 1,
+        'return' => array("id"),
+        'name_a_b' => "Line Manager is",
+      ));
+      $relationshipTypeID = $result['id'];
+      $queryParam = array(1 => array($contactID, 'Integer'),
+        2 => array($relationshipTypeID, 'Integer'));
```

Comment:
I think this would be better formatted as:
```
array(
  1 => array($contactID, 'Integer'),
  2 => array($relationshipTypeID, 'Integer')
);
```

The way it's formatted, looks like they're two separate arrays.